x-pack/plugins/observability/server/utils/create_or_update_index.ts

```diff
@@ -64,26 +70,28 @@ export async function createOrUpdateIndex({
   }
 }
 
-function createNewIndex({
+async function createNewIndex({
   index,
   client,
   mappings,
+  settings,
 }: {
   index: string;
   client: ElasticsearchClient;
   mappings: Required<estypes.IndicesCreateRequest>['body']['mappings'];
+  settings: Required<estypes.IndicesPutSettingsRequest>['body']['settings'];
 }) {
-  return client.indices.create({
+  return await client.indices.create({
```

Comment:
Why do we need to await here instead of just passing the promise?

Reply:
Not sure if that was the intention here, but awaiting a promise inside of a function can be useful for debugging, as it ensures the function shows up in the stacktrace. Here's some explanation: https://github.com/goldbergyoni/nodebestpractices/issues/737